Makefile

```diff
@@ -80,7 +80,7 @@ ifdef CONFIG_SHARED
     LDFLAGS_DLL = $(LDFLAGS) -shared
 
     ifdef CONFIG_WINDOWS
-        IMP_LIB = libdcadec/libdcadecdll$(LIBSUF)
+        IMP_LIB = libdcadec/libdcadec.dll$(LIBSUF)
```

Comment:
It's cosmetics. I don't want to redo the pull request unless there's a problem, and it can always be changed later.